snuba/migrations/runner.py

```diff
@@ -390,6 +390,67 @@ def get_status(migration_key: MigrationKey) -> Status:
         else:
             self._reverse_migration_impl(migration_key)
 
+    def reverse_all(
+        self,
+        *,
+        through: str = "all",
+        fake: bool = False,
+        force: bool = False,
+        include_system: bool = False,
+        group: Optional[MigrationGroup] = None,
+        readiness_states: Optional[Sequence[ReadinessState]] = None,
+    ) -> None:
+
+        groups = (
+            [group]
+            if group
+            else (
+                get_active_migration_groups()
+                if include_system
+                else [
+                    g
+                    for g in get_active_migration_groups()
+                    if g != MigrationGroup.SYSTEM
+                ]
+            )
+        )
+        completed_migrations = self._get_completed_migrations(groups)
+
+        if readiness_states:
+            completed_migrations = [
+                m
+                for m in completed_migrations
+                if get_group_readiness_state(m.group) in readiness_states
+            ]
+
+        use_through = False if through == "all" else True
+
+        def exact_migration_exists(through: str) -> bool:
+            migration_ids = [
+                key.migration_id
+                for key in completed_migrations
+                if key.migration_id.startswith(through)
+            ]
+            if len(migration_ids) == 1:
+                return True
+            return False
```

Comment:
Should this be renamed to `exact_one_migration_exists`?
```
def exact_one_migration_exists(through: str) -> bool:
    return sum(1 for key in completed_migrations if key.migration_id.startswith(through)) == 1
```

Reply:
I think the naming is okay imo, but I will simplify this a bit since we can just return `len(migration_ids) == 1`